gloo/cuda_allreduce_ring_chunked.cc

```diff
@@ -71,7 +71,11 @@ CudaAllreduceRingChunked<T, W>::CudaAllreduceRingChunked(
   // (256 * sizeof(float)).
   constexpr unsigned long minSize = 256;
```

Comment:
Same as above, just make this explicitly an `unsigned long long`. `long` is 64-bit on linux, but 32-bit on windows, so it's far better to make this explicit. (I would recommend searching through gloo for any references to bare `long` and fixing them, otherwise you're likely to run into issues later)